pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib, stdenv, fetchgit, pkgconfig, cmake, gnumake
+, enableJpeg ? false, libjpeg
+, enableZlib ? false, zlib
+, enablePng ? false, libpng
+, enableEigen ? false, eigen
+, enableTiff ? false, libtiff
+, enableExamples ? false
+, enableDocs ? false }:
+
+stdenv.mkDerivation rec {
+  version = "1.1";
+  name = "openmvg-${version}";
+
+  src = fetchgit {
+    url = "https://www.github.com/openmvg/openmvg.git";
+
+    # Tag v1.1
+    rev = "f5ecb48";
+    sha256 = "1di9i7yxnkdvl8lhflynmqw62gaxwv00r1sd7nzzs9qn63g0af0f";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = 
+       [ cmake ]
+    ++ lib.optional enableJpeg libjpeg
+    ++ lib.optional enableZlib zlib
+    ++ lib.optional enablePng libpng
+    ++ lib.optional enableEigen eigen
+    ++ lib.optional enableTiff libtiff
+    ;
+
+  nativeBuildInputs = [ gnumake pkgconfig ];
+
+  cmakeFlags = [
+    "-DOpenMVG_BUILD_EXAMPLES=${if enableExamples then "ON" else "OFF"}"
+    "-DOpenMVG_BUILD_DOC=${if enableDocs then "ON" else "OFF"}"
+    "-DCMAKE_CXX_FLAGS=-std=c++11"
+  ];
```

Comment:
Well, adding five parameters and later handling them just to do the same as a single comment would accomplish seems too verbose to me.